curation_portal/migrations/0015_usersettings_ucsc_session_name_grch38.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.24 on 2023-04-27 03:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
This was me. Wasn't sure if django would treat this as a column rename and an add column operation, or if it would treat it as a delete column and add two new columns operation. I wanted the former so I specifically split it into two migrations.